backend/src/test/java/gov/cdc/usds/simplereport/service/email/EmailServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package gov.cdc.usds.simplereport.service.email;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.sendgrid.helpers.mail.Mail;
+import gov.cdc.usds.simplereport.service.BaseServiceTest;
+import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration.WithSimpleReportSiteAdminUser;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+class EmailServiceTest extends BaseServiceTest<EmailService> {
+  @MockBean EmailProviderWrapper mockSendGrid;
+
+  @Autowired EmailService _emailService;
```

Comment:
If you do stick with extending `BaseServiceTest`, this is unnecessary—you can just use `_service` (it's autowired in the base class).